python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -308,21 +308,22 @@ class RandomForestRegressor(Base):
         cdef size_t params_t64
         if self.n_cols:
             # only if model has been fit previously
-            model_pbuf_bytes = self._get_protobuf_bytes()
-            params_t = <uintptr_t> self.rf_forest
-            rf_forest = \
-                <RandomForestMetaData[float, float]*>params_t
-            params_t64 = <uintptr_t> self.rf_forest64
-            rf_forest64 = \
-                <RandomForestMetaData[double, double]*>params_t64
-            state["rf_params"] = rf_forest.rf_params
-            state["rf_params64"] = rf_forest64.rf_params
-        else:
-            model_pbuf_bytes = bytearray()
+            self._get_protobuf_bytes()  # Ensure we have this cached
```

Comment:
Can we consolidate this between classifier and regressor? I notice the code is identical between the two

Reply:
As per the above, the datatypes differ, so it's a bit tricky. Ugh! Sadly, I think an 0.15 thing.